test/e2e/runner/perturb.go

```diff
@@ -48,11 +44,11 @@ func PerturbNode(node *e2e.Node, perturbation e2e.Perturbation) (*rpctypes.Resul
 	switch perturbation {
 	case e2e.PerturbationDisconnect:
 		logger.Info("perturb node", "msg", log.NewLazySprintf("Disconnecting node %v...", node.Name))
-		if err := execDocker("network", "disconnect", testnetName, name); err != nil {
+		if err := execDocker("network", "disconnect", testnet.Name+"_"+testnet.Name, name); err != nil {
 			return nil, err
 		}
 		time.Sleep(10 * time.Second)
-		if err := execDocker("network", "connect", testnetName, name); err != nil {
+		if err := execDocker("network", "connect", testnet.Name+"_"+testnet.Name, name); err != nil {
```

Comment:
Same as above

```suggestion
		if err := execDocker("network", "connect", testnet.Name, name); err != nil {
```

Reply:
I ended up just deferring to upstream

agree that this is weird